### ISSUE TITLE
In-house Forums As defined by #694

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/MyopicVicar#1355]
Forums (1.6)
Some permanent forums (a ‘queries on transcriptions’ group, for example), some temporary (e.g. to discuss Transcriber Agreement).
‘Opt in’ participation – people can select to get no email alerts, alert for each forum post, daily or weekly summary, daily or weekly compilation. Forum posts only visible once logged in – forum may be locked to specific roles or individuals (e.g. County Coordinators only).
Some forums might link well to social media – e.g. posting a question might tweet the subject line, or post to Facebook. This might be particularly good for engaging new volunteers, and promoting the resources we have available, but some monitoring would be needed, and possibly even feeding back from social media to the forums.
‘How to’ for best practice on forums, and inclusion in Codes of Conduct.
Posts listed on website in order of most recent contribution.
Number of responses to posts visible.
Responses ‘threaded’ to original post.
Response via the website or via email routed through the website.
Original poster, respondent and moderators can ‘close’ threads.

E3 U3 S6 C?3: P15

**Comment by PatReynolds** [comment]
Duplicate of #693

**Comment by Captainkirkdawson** [comment]
I am hoping you are not proposing that we write our own forum system and if you are not what are you proposing?  
We already use too many communications systems what with tech lists, slack, waffle and multiple google groups

**Comment by PatReynolds** [comment]
Yes, to replace google Q+A, google groups in FreeREG (and rootsweb in FreeBMD and FreeCEN), and additional groups (e.g. for researchers to ask questions).  To be discussed at next TASc meeting.  Using Mailman now seems an alternative (but I can't find out much about it).

**Comment by Captainkirkdawson** [comment]
Personally do not see Mailman as an alternative! 
Nor would I support yet another application coming in-house. We cannot support those we have. 
Will comment during TASc

**Comment by PatReynolds** [comment]
I went back to the original specification and copied and pasted into the description.  See comment above about in-house (original vision).  Google groups seem fine as a forum alternative, so suggest this is considered as an epic (e.g. on creating new forums, documentation, etc.)

**Comment by edickens** [comment]
The problem with Google Groups is that you have to manually invite or add every new person who registers.  And then they can delete themselves.  So you do not have a mailing system that reaches every active transcriber.
The FreeREG Transcribers group is a closed group so that we don't get spam, and I invite everyone, but less than half take up the invitation.

**Comment by PatReynolds** [comment]
Hi Eric, this story is not about the mailing system that reaches every transcriber.  It is about discrete groups (such as the existing one for the development team).  People who don't want to be part of the group/team simply won't join, or will resign if they don't want to participate for any reason, or put the emails on hold if it is a temporary withdrawal (eg because they are away from home).

**Comment by Captainkirkdawson** [comment]
I see no point in developing forum software for FreeREG. There exist excellent forum capabilities through Google Groups. 
We can create as many or as few as we wish. They met all of the requirements identified. If you really do NOT want to use Google Groups for some reason then it would be possible to host fora through other suppliers or as part FreeUKGEN. Should FreeUKGEN be migrated to WordPress then there are excellent forum extensions for it. 
Personally I see little benefit and extensive costs associated with bringing fora in house even using WordPress. 
A minimum cost solution if you feel there are weaknesses might be for our engagements coordinator to take all existing fora both in Google Groups and else where under a central control adding and culling groups as management sees fit.
As you know I find the costs of participating in any fora too high for any benefit.
The Transcribers group works well. 
If we need to send messages to all Members or groups of Members then we do have the means of so doing.

**Comment by Captainkirkdawson** [comment]
@arswright-alt-uk may wish to comment

**Comment by stsccfr** [comment]
Closing as being way more than we can ever hope to implement and manage in-house
